drivers/CoreTouchSensor/include/CoreTouchSensor.h

```diff
@@ -0,0 +1,43 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "drivers/interfaces/InterfaceDigitalIn.h"
+#include "drivers/interfaces/InterfaceDigitalOut.h"
+
+#include "interface/drivers/QDAC.h"
+
+namespace leka {
+
+enum class PowerMode : uint8_t
+{
+	low,
+	normal
+};
+
+class CoreTouchSensor
+{
+  public:
+	explicit CoreTouchSensor(mbed::interface::DigitalIn &detect_pin, mbed::interface::DigitalOut &power_mode_pin,
+							 interface::QDAC &dac, Channel channel)
+		: _detect_pin(detect_pin), _power_mode_pin(power_mode_pin), _sensitivity_pin({dac, channel})
+	{
+		_detect_pin.mode(PinMode::PullUp);
+		_sensitivity_pin.dac.init();
+	}
+	auto read() -> int;
+	void setPowerMode(PowerMode power_mode);
+	void adjustSensitivity(uint16_t value, bool saved = false);
```

Comment:
Utilises `set` plutôt que `adjust`, c'est plus standard et elle est absolue là où ajuster c'est une variation par relatif à un point de départ (ajuster un peu sur la droite, un peu sur la gauche, ...)

```suggestion
	void setSensitivity(uint16_t value, bool saved = false);
```